INSTALL.rst.txt

```diff
@@ -61,6 +61,8 @@ To perform an inplace build that can be run from the source folder run::
 
     python setup.py build_ext --inplace -j 4
 
+Note that the ``python`` command is for installation on python 2. For installation on python 3, replace with the ``python3``command. See `Requirements for Installing Packages <https://packaging.python.org/tutorials/installing-packages/>`_ for more details.
```

Comment:
Missing space before "command"

Also, this isn't true. `python` is for installation on whatever version of python `python` is. I don't think it's the place of the numpy docs to tell users what `python` means on their system - but maybe we can tell them to check.

Reply:
It might be worth mentioning that for most Linuces `python` is Python 2 - unless you're in a virtualenv.

This is true for Debian / Ubuntu, Fedora, appears to be true for Mint, and isn't true for Arch (`python` is Python 3 by default - https://wiki.archlinux.org/index.php/Python).

Reply:
@matthew-brett I think that will change. Fedora is planning on making the change with Fedora 32 ~= first half of 2020.